app/views/shared/_nav_menu_header.html.erb

```diff
@@ -0,0 +1,10 @@
+<header class="w-full p-4">
+  <div class="flex flex-wrap items-center justify-end gap-4 w-10/12 mx-auto">
+    <% if user_signed_in? %>
+      <%= link_to "Sign out", destroy_user_session_path, class: 'px-4 py-2 border-2 border-black cursor:pointer', method: :delete %>
```

Comment:
- [ ] I got the error on the screenshot below after clicking on this link to sign out. Let's kindly fix it.
![Screenshot from 2023-08-31 01-37-36](https://github.com/lordksix/blog-by-rails/assets/63087491/2fba3130-875d-45d7-87b6-f399df796784)

Reply:
Thank you for the feedback. The issues has been resolved